models/anderson_2007.cpp

```diff
@@ -2,18 +2,27 @@
 // from anderson et al 2007 - Vancomycin pharmacokinetics in preterm neonates and 
 // the prediction of adult clearance.
 [PARAM] @annotated
-CL    : 3.83 : Clearance (L/hr)
-V     : 39.4 : Volume (L) 
-WT    : 2.9  : Weight (kg)
-PMA   : 34.8 : post-menstral age (wk)
-SCR   : 0.9  : serum creatinine (mg/dL)
-VENT  : 0    : Ventillation status
-IONT  : 0    : ionotrope status
-Kage  : 0.00789 : scaling constant of age on renal function
-Fvent : 1.03 : Scaling factor applied for use of positive pressure artificial ventilation
-Fiont : 1.19 : Scaling factor applied for use of inotropes (use of dopamine)
-OCC   : 1     : Occasion
-  
+CL        : 3.83    : Clearance (L/h)
+V         : 39.4    : Volume (L) 
+WT        : 2.9     : Weight (kg)
```

Comment:
The term "normalize" has no standard meaning that I am aware of. I try to avoid it but if can be useful e.g. to identify what is the weight and height and renal function of a healthy full term neonate at birth (a healthy "normal" baby at birth - NBAB) so that those who have not thought much about the subject have a point of reference.

The term "typical" has been used in the pop PK literature to describe a group the parameter associated with the same covariates (weight, height, renal function, age). I prefer to describe this as the group value which is derived from the population standard value given a particular group of covariates. Individual values are then derived from group values by adding a random effect. That is why the DDMoRe MDL language has population, group and individual parameter values.
For your application I suggest you try to identify a set of covariates that describe a healthy full term baby at birth. A well constructed model will be able to predict the parameter of interest in a NBAB from the standard covariate values. That NBAB changes rapidly as the days pass. A well constructed model should be able to predict these changes. "local" models that cannot extrapolate to other humans have limited value and I would not put much effort into describing them. Better to get the original data and use it to develop a "global" model.

This is my goal for covariate based models. I don't have all the answers today but theory based allometric scaling plus a sigmoid emax maturation model provide a consistent basis for a "standard PK model" that works quite well for the NBAB (Holford NHG, Anderson BJ. Why standards are useful for predicting doses. Br J Clin Pharmacol. 2017;83(4):685-7). You might want to adopt that until someone comes up with something better.

Do you have any overall plan for why you are coding models such as vancomycin PK in neonates?